src/network/graphMutation.js

```diff
@@ -43,7 +46,9 @@ class GraphMutation {
 
   async execute() {
     const mutation = this.build();
-    console.debug(mutation);
+    if (process.env.REACT_APP_ENV === 'dev') {
+      window.mutations += `\n${mutation}`;
```

Comment:
I think it should be fine. Not just because this is only in the dev environment, but in [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length) it says `ECMASCript 2016 (ed. 7) established a maximum length of 2^53 - 1 elements. Previously, no maximum length was specified. `

Reply:
👍 just wondering. this shouldnt be a problem since we care about the newest logs usually.